src/spine.js

```diff
@@ -58,32 +58,38 @@ class Spine {
 				item.canonical = canonical(item.href);
 			}
 
-			if(manifestItem) {
+			if (manifestItem) {
+				//KEM: This is where spine items get the URL from the manifest, this is where to connect the manifest media-overlay and get SMIL URL
 				item.href = manifestItem.href;
 				item.url = resolver(item.href, true);
 				item.canonical = canonical(item.href);
-
-				if(manifestItem.properties.length){
+				//KEM: added overlay so can load later in section
```

Comment:
The change here is to load the smil information from the manifest into the saved list of items that keeps track of the page urls and properties.  Now the smil will be associated with the page, and this is used later in section.js when the page is loaded and displayed.